src/DummyBackendBase.cc

```diff
@@ -37,4 +38,34 @@ namespace ChimeraTK {
     }
   }
 
+  template<typename UserType>
```

Comment:
I did not check that this is the code moved from the header file.

Reply:
In the diff, this code block is "new", but in fact, it has been moved here from the header. I did not compare the old header and the new .cc file to check that this section is identical in both.

Reply:
Ah. There is the change to properly chain up of course, but otherwise it should be a move